modules/gapi/src/backends/common/serialization.cpp

```diff
@@ -366,6 +366,17 @@ IIStream& operator>> (IIStream& is,       cv::gapi::wip::draw::Line &l) {
     return is >> l.color >> l.lt >> l.pt1 >> l.pt2 >> l.shift >> l.thick;
 }
 
+IIStream& operator>> (IIStream& is, cv::gapi::ot::TrackingStatus& status) {
+    std::string statusStr;
+    is >> statusStr;
+    status = cv::gapi::ot::from_string(statusStr);
+    return is;
```

Comment:
Please don't do this. It is binary serialization, no the formatted input/output

Reply:
Removed!